packages/plugins/plugin-deck/src/components/DeckLayout/ComplementarySidebar.tsx

```diff
@@ -18,41 +19,45 @@ import { useNode, useNodeActionExpander } from '../../hooks';
 import { useLayout } from '../LayoutContext';
 
 export type ComplementarySidebarProps = {
-  id?: string;
+  context?: string;
   layoutParts: LayoutParts;
   flatDeck?: boolean;
 };
 
-export const ComplementarySidebar = ({ id, layoutParts, flatDeck }: ComplementarySidebarProps) => {
+export const ComplementarySidebar = ({ context, layoutParts, flatDeck }: ComplementarySidebarProps) => {
   const { popoverAnchorId } = useLayout();
+  const attentionPlugin = useResolvePlugin(parseAttentionPlugin);
+  const attention = attentionPlugin?.provides.attention;
+  const attended = attention?.attended ? Array.from(attention.attended) : [];
```

Comment:
Let's make a hook called `useAttendedIds`. I've seen this repeated a few times now. 
Could put it here: https://github.com/dxos/dxos/blob/0973be1883a944ea2d1caa8e2515e7eaddef8fc0/packages/ui/react-ui-attention/src/components/AttentionProvider.tsx#L33